GITHUB_ACTIONS_TO_HARNESS_CI/github_actions_to_harness_ci.md

```diff
@@ -0,0 +1,224 @@
+# Migration Guide - Github Actions to Harness-CI
+
+## Introduction
+Harness CI and GitHub Actions both allow you to create workflows that automatically build, test, publish, release, and deploy code. Harness CI and GitHub Actions share some similarities in workflow configuration:
+
+*   Workflow configuration files are written in YAML and stored in the repository.
+*   Workflows include one or more stages/jobs.
+*   Stages include one or more steps or individual commands.
+*   Steps or tasks can be reused and shared with the community.
+
+## Key Differences 
+* With Harness, there’s no scripting needed and configurations are passed to pipelines securely and in a pragmatic way while Github Actions has third-party actions that you can use as semi-plug-and-play functionality.
+* HashiCorp created an action to set up and configure the Terraform CLI in the GitHub Actions workflow. There’s also an action for CloudFormation. Harness provides both infrastructure provisioners with a simpler structure and configuration.
+* GitHub Actions does not provide a native Accelerate metrics dashboard whereas Harness has a dashboard specifically for these metrics and allows you to set alerts as needed.
+* The YAML file for Github actions is stored in the .github/workflows folder in a repository and for Harness CI it’s stored on the Harness itself and can be created from UI or by importing it from a Git Source.
+
+## Comparison 
+
+### 1. Define a stage that executes a single build step
+
+- Github Actions
+ 
+         jobs:
+          build_test_and_run:
+            name: build test and run 
+
+- Harness CI 
+        
+        stages:
+        -stage:
```

Comment:
```suggestion
         stage:
```
not sure if this is meant to be here but throws a lint error while trying a YAML linter.

Reply:
hi @hrittikhere , there was some indentation thing , fixed it .Thanks .
It will be written like that only , it's just a space was required.